pkg/build/builder/sti.go

```diff
@@ -227,6 +227,13 @@ func (s *S2IBuilder) Build() error {
 		config.RuntimeImage = runtimeImageName
 		config.RuntimeAuthentication, _ = dockercfg.NewHelper().GetDockerAuth(runtimeImageName, dockercfg.PullAuthType)
 		config.RuntimeArtifacts = copyToVolumeList(s.build.Spec.Strategy.SourceStrategy.RuntimeArtifacts)
+		if s.build.Spec.Strategy.SourceStrategy.ForcePull {
+			glog.V(4).Infof("With force pull true, setting runtime image pull policies to %s", s2iapi.PullAlways)
+			config.RuntimeImagePullPolicy = s2iapi.PullAlways
+		} else {
+			glog.V(4).Infof("With force pull false, setting runtime image pull policies to %s", s2iapi.PullIfNotPresent)
+			config.RuntimeImagePullPolicy = s2iapi.PullIfNotPresent
+		}
```

Comment:
I would rather log the message once:
```go
glog.V(4).Infof("With force pull %v, setting runtime image pull policy to %s", s.build.Spec.Strategy.SourceStrategy.ForcePull, s2iapi.PullIfNotPresent)
```

Also s/policies/policy/ in the message.

BTW, do we really need this message at all?